### PR TITLE
Fixing Issue #133

### PR DESCRIPTION
**Pull request by SimplicityGuy**
Addresses Issue #133

### CODE REVIEW

File: usb/backend/libusb0.py [hunk 1]
```diff
@@ -487,10 +493,6 @@ def set_configuration(self, dev_handle, config_value):
         _check(_lib.usb_set_configuration(dev_handle, config_value))
 
     @methodtrace(_logger)
-    def set_interface_altsetting(self, dev_handle, intf, altsetting):
-        _check(_lib.usb_set_altinterface(dev_handle, altsetting))
-
-    @methodtrace(_logger)
     def get_configuration(self, dev_handle):
```
Comment (walac) [inline usb/backend/libusb0.py]: Why was this changed?

File: usb/backend/libusb0.py [hunk 2]
```diff
@@ -510,6 +512,9 @@ def get_configuration(self, dev_handle):
         assert ret == 1
         return buff[0]
 
+    @methodtrace(_logger)
+    def set_interface_altsetting(self, dev_handle, intf, altsetting):
+        _check(_lib.usb_set_altinterface(dev_handle, altsetting))
```
Comment (walac) [inline usb/backend/libusb0.py]: Same here, any reason to changes here?

File: usb/backend/libusb0.py (hunk 1 above)
Comment (SimplicityGuy) [inline usb/backend/libusb0.py]: I moved it so that the `libusb0` implements `IBackend` in the same order as the `IBackend` interface is defined. Not strictly needed, just easier for readability when comparing `libusb0` vs `libusb1` vs `IBackend`.

File: usb/backend/libusb0.py (hunk 2 above)
Comment (SimplicityGuy) [inline usb/backend/libusb0.py]: See previous comment; just moved the implementation of this to match the definition of the interface.

File: usb/backend/libusb0.py (hunk 1 above)
Comment (walac) [inline usb/backend/libusb0.py]: Ok, but notice you accidentally removed `@methodtrace` from `get_configuration`

File: usb/backend/libusb0.py (hunk 2 above)
Comment (SimplicityGuy) [inline usb/backend/libusb0.py]: It's still there... Sadly the diff in github isn't great. See line 489/495 (old/new).

File: usb/backend/libusb0.py (hunk 1 above)
Comment (SimplicityGuy) [inline usb/backend/libusb0.py]: It's still there... Sadly the diff in github isn't great. See line 489/495 (old/new).

**Comment by SimplicityGuy** [comment]
Just a friendly ping on this request...

**Comment by walac** [comment]
Sorry, I forgot to merge it. Thanks for your contribution.
